### ISSUE TITLE
Add short names for events

### ISSUE THREAD

**Issue by mbollmann** [acl-org/acl-anthology#791]
> Can I piggyback on this and request that this YAML file include an "official" short name for every event? For example, P18 would be "ACL 2018", but P08 would be "ACL-08: HLT".

_Originally posted by @davidweichiang in https://github.com/acl-org/acl-anthology/issues/149#issuecomment-467044062_

This will be facilitated by the restructuring of YAML files as suggested in #790.

**Comment by mbollmann** [comment]
> My thought for short names was to put them in the XML, as a `<shortbooktitle>` tag, next to `<title>` in the `<meta>` block.

_Originally posted by @mjpost in https://github.com/acl-org/acl-anthology/pull/790#issuecomment-615953684_

**Comment by akoehn** [comment]
The new ID schema has human-readable IDs, which could be used in conjunction with the year.  Maybe that is enough information?

**Comment by mjpost** [comment]
That's a good thought. Using the acronym from the venues file (which is the cased version of the key) could be sufficient, e.g., "Proceedings of {acronym} {year}".

**Comment by mbollmann** [comment]
But this does not address the original motivation for doing this, namely conferences sometimes having their own non-obvious acronym, like ACL-08: HLT.

**Comment by akoehn** [comment]
That is correct. I just thought that an approximate version is better than none. To be fair, I don't really care about short names here and also don't know what exactly should be done with them -- I would oppose a second bibtex button on each publication for "short" citations.

So my main motivation was to close some stale issues by either finding a solution or deciding that we do not actually need a solution ;-)

**Comment by mbollmann** [comment]
Since it was @davidweichiang who suggested and also offered to help with that, maybe he can comment if that is still the case. :)

When this originally came up, it wasn't clear yet where this info would even go, so nothing concrete happened, but I think it is clear now.

**Comment by davidweichiang** [comment]
How about if the default short name is `Proceedings of {acronym} {year}` but can be overridden with `<shortbooktitle>`?

**Comment by akoehn** [comment]
David Chiang writes:

> How about if the default short name is `Proceedings of {acronym} {year}` but can be overridden with `<shortbooktitle>`?

That is what I would do but I still don’t know *where* this is supposed
to be used.

**Comment by davidweichiang** [comment]
If we don't want short names to be the default, I believe the short names would be used nowhere. It would be up to users of the bib files to modify them to use the short names instead.

I suppose someone (@nschneid?) could make a bst file that uses the short names.

**Comment by mjpost** [comment]
I haven't looked into details, but it seems possible to dump a separate BibTeX field (`shortbooktitle`). We could then also alter the style files to optionally use this field instead of `booktitle` when generating the reference list, e.g.,

```latex
\usepackage{hyperref,shorttitles}[acl2021]
```

**Comment by nschneid** [comment]
I'm not a BibTeX wizard but having a separate field sounds good to me!

Zotero has a "Conference Name" field which the importer is currently filling in heuristically.

**Comment by mbollmann** [comment]
Non-standard BibTeX fields might get cut out by reference managers though, and I don't believe there's a standard field that fits this information.

**Comment by nschneid** [comment]
Is this still relevant? Maybe we should avoid introducing too much variability in how papers are cited.

**Comment by nschneid** [comment]
We have a way to generate short venue names for citations so I am going to close this.
